packages/docs/pages/components/button.mdx

```diff
@@ -0,0 +1,118 @@
+import Button from '../../components/Button/Button';
+
+# Bottons
+
+Buttons allow users to take actions, and make choices, with a single tap.
+
+## Basic Buttons
+
+The `button` has two variants: `Primary` and `Secondary`.
+
+<div style={{ display: 'grid', gridTemplateColumns: 'auto auto', justifyContent: 'center' }}>
+  <Button>Primary</Button>
+  <Button variant="secondary">Secondary</Button>
+</div>
+
+### Primary Button
+
+`Pramry Button` are high-emphasis, is used to actions primary of you app.
+
+```jsx
+import { Button } from '@platzily-ui/components';
+import { Fragment } from 'react';
+
+const MyComponent = () => {
+  return (
+    <Fragment>
+      <Button>Normal</Button>
+      <Button disabled>Disabled</Button>
+    </Fragment>
+  );
+};
+```
+
+<div style={{ display: 'grid', gridTemplateColumns: 'auto auto', justifyContent: 'center' }}>
+  <Button>Normal</Button>
+  <Button disabled>Disabled</Button>
+</div>
+
+### Secondary Button
+
+`Secondary Button` are medium-emphasis, is used to actions secondary of you app.
+
+```jsx
+import { Button } from '@platzily-ui/components';
+import { Fragment } from 'react';
+
+const MyComponent = () => {
+  return (
+    <Fragment>
+      <Button variant="secondary">Normal</Button>
+      <Button variant="secondary" disabled>
+        Disabled
+      </Button>
+    </Fragment>
+  );
+};
+```
+
+<div style={{ display: 'grid', gridTemplateColumns: 'auto auto', justifyContent: 'center' }}>
+  <Button variant="secondary">Normal</Button>
+  <Button variant="secondary" disabled>
+    Disabled
+  </Button>
+</div>
+
+## Handling
+
+All components accept handlers that is applied to the root DOM element.
+
+```jsx
+import { Button } from '@platzily-ui/components';
+import { Fragment } from 'react';
+
+const MyComponent = () => {
+  const handleClick = () => {
+    alert('clicked');
+  };
+
+  return (
+    <Fragment>
+      <Button onClick={handleClick}>Click me</Button>
+    </Fragment>
+  );
+};
+```
+
+## Colors
+
+In addition to using the default button colors, you can add custom. Allowed format: `'rgb', 'rgba', 'hsl', 'hsla', 'Hex'`
+
+```jsx
+import { Button } from '@platzily-ui/components';
+import { Fragment } from 'react';
+
+const MyComponent = () => {
+  const handleClick = () => {
+    alert('clicked');
+  };
+
+  return (
+    <Fragment>
+      <Button color="#FFC0CB">
```

Comment:
I don't think I like this approach, because the button handle multiple colors, for example, the font color, the background color, the background color when hover and active. So, don't you think if we're going with this approach we should support all of them?